.gitignore

```diff
@@ -1,2 +1,4 @@
 _site/
 .DS_Store
+.jekyll-metadata
```

Comment:
Sounds good, I figured everyone would want gitignore these random build cache files, but I'll just go back to default.